src/main/java/gregtech/common/covers/CoverBehaviors.java

```diff
@@ -61,6 +61,16 @@ public static void init() {
         registerBehavior(37, new ResourceLocation(GTValues.MODID, "machine_controller"), MetaItems.COVER_MACHINE_CONTROLLER, CoverMachineController::new);
         registerBehavior(38, new ResourceLocation(GTValues.MODID, "smart_filter"), MetaItems.SMART_FILTER, (tile, side) -> new CoverItemFilter(tile, side, "cover.smart_item_filter.title", Textures.SMART_FILTER_FILTER_OVERLAY, new SmartItemFilter()));
         registerBehavior(39, new ResourceLocation(GTValues.MODID, "facade"), MetaItems.COVER_FACADE, CoverFacade::new);
+
+        registerBehavior(40, new ResourceLocation(GTValues.MODID, "fluid.regulator.lv"), MetaItems.FLUID_REGULATOR_LV, (tile, side) -> new CoverFluidRegulator(tile, side, GTValues.LV, 1280));
+        registerBehavior(41, new ResourceLocation(GTValues.MODID, "fluid.regulator.mv"), MetaItems.FLUID_REGULATOR_MV, (tile, side) -> new CoverFluidRegulator(tile, side, GTValues.MV, 5120));
```

Comment:
Maybe we could invest to system which will create these for us more easily as we can take names from array we could have all of it in arrays and just run it on correct set of them.

Given that you need these arrays in MachineRecipeLoader anyway.

What you think?

Reply:
Yes, this sounds like a good idea. I'm not sure where I should place this array, what do you think?

Reply:
I don't feel like this has API potential so we should leave it in common. Maybe something like `GTValuesCommon` as we may store there later more of these.